app/views/users/_account_set_password.html.erb

```diff
@@ -9,7 +9,7 @@
         If you signed up with a social media account, please <a href="<%= new_user_password_path %>">reset the password</a> for your primary email address (<%= current_user.email %>) first.
       </p>
     </header>
-    <%= form_with url: user_update_password_path do |f| %>
+    <%= form_with(url: user_update_password_path, data: { testid: "update-password-form" }) do |f| %>
```

Comment:
Required for end to end tests to find the update password form reliably. See [Making your UI tests resilient to change](https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change).